### PR TITLE
Master

### PR DESCRIPTION
**Pull request by ygavish**
---
Please check the following conditions for your PR.

- [x] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [x] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

File: certified-connectors/Projectplace/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,1190 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",
```
Comment (dougelliott) [inline certified-connectors/Projectplace/apiDefinition.swagger.json:4]: Title needs to be updated

**Review by dougelliott — CHANGES_REQUESTED** [review]

File: certified-connectors/Projectplace/apiDefinition.swagger.json (hunk 1 above)
Comment (sriyen-msft) [reply certified-connectors/Projectplace/apiDefinition.swagger.json:4]: @ygavish Please refer [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#submission-requirements)

File: certified-connectors/Projectplace/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,1190 @@
...
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "project_id"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "list_boards",
```
Comment (sriyen-msft) [inline certified-connectors/Projectplace/apiDefinition.swagger.json:1062]: @ygavish This operation may not be visible to public, but still the summary and description values should be as per standards, please provide a valid sentence.

Refer [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

File: certified-connectors/Projectplace/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,1190 @@
...
+                  "name": {
+                    "type": "string",
+                    "description": "name"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "get_planlets",
```
Comment (sriyen-msft) [inline certified-connectors/Projectplace/apiDefinition.swagger.json:1093]: @ygavish This operation may not be visible to public, but still the summary and description values should match the standards, please provide a valid sentence.

Refer [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

File: certified-connectors/Projectplace/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,1190 @@
...
+                  "type": {
+                    "type": "string",
+                    "description": "type"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "list_members",
```
Comment (sriyen-msft) [inline certified-connectors/Projectplace/apiDefinition.swagger.json:1156]: @ygavish Same here

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@ygavish, Can you please remove other connector files in the PR.

**Comment by ygavish** [comment]
Thank you for your remarks. The PR is now updated.

File: certified-connectors/Projectplace/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,1190 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Projectplace",
+    "description": "Get work done across projects and teams\nProjectplace lets you plan, collaborate and track progress of all projects and assignments in an easy and user-friendly way.",
```
Comment (dougelliott) [inline certified-connectors/Projectplace/apiDefinition.swagger.json:5]: Could this all be one line? So `Get work done across projects and teams. Projectplace lets you plan, collaborate...` That would help with readability in certain parts of the platform.

**Review by dougelliott — CHANGES_REQUESTED** [review]

File: certified-connectors/Projectplace/apiDefinition.swagger.json (hunk 5 above)
Comment (ygavish) [reply certified-connectors/Projectplace/apiDefinition.swagger.json]: Changed to one line

**Review by dougelliott — APPROVED** [review]

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**